### PR TITLE
(DOCSP-22940): Created new Realm Web SDK Install Page and Updated

### PR DESCRIPTION
**Pull request by chloe-ratte**
## Pull Request Info

Updated Realm Web SDK page to reflect how and when Node.js can be used with Realm Web SDK. Also added Install page for Web which explains the general and for Node.js installation processes respectively.

### Jira

- https://jira.mongodb.org/browse/DOCSP-22940

### Staged Changes (Requires MongoDB Corp SSO)

- [Realm Web SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-22940/web)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/web/web-install.txt [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. _web-install:
```
Comment (mongodben) [inline source/web/web-install.txt:1]: rename the file to be just 'install.txt', and update TOC tree directive in `source/web.txt` accordingly

File: source/web/web-install.txt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+=====================
+Install the Realm Web SDK
+=====================
```
Comment (mongodben) [inline source/web/web-install.txt:5]:
```suggestion
=========================
Install the Realm Web SDK
=========================
```

File: source/web/web-install.txt [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+.. default-domain:: mongodb
```
Comment (mongodben) [inline source/web/web-install.txt:7]:
```suggestion
```

we actually don't need this. it's a vestige of earlier versions of docs platform.
we haven't bothered deleting from old pages, but don't add to new

File: source/web/web-install.txt [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This page shows you how to install the Realm Web SDK to use with your application. 
+Before you begin, you'll need to :ref:`create an {+app+} <create-a-realm-app>` for 
+your web app to use.
```
Comment (mongodben) [inline source/web/web-install.txt:19]: i would advise making this it's own section, like https://www.mongodb.com/docs/realm/sdk/flutter/sync/open-synced-realm/#before-you-begin

or at least make it it's own paragraph

File: source/web.txt [hunk 5]
```diff
@@ -27,11 +28,9 @@ Atlas.
...
-   The Web SDK does not support JavaScript or TypeScript applications
-   written for the Node.js (server-side) environment or React Native
-   mobile apps. For those use cases, you should consider the
-   :ref:`Node.js SDK <node-intro>` and the :ref:`React Native SDK
-   <react-native-intro>`.
-
-.. button:: Create an Account
-   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_realm_web
+   There are SDKs built for :ref:`React Native <react-native-intro>`
+   and :ref:`Node.js <node-intro>` which should be used when creating
```
Comment (mongodben) [inline source/web.txt:32]: > "should be used "

rephrase to avoid passive voice

File: source/web.txt [hunk 6]
```diff
@@ -27,11 +28,9 @@ Atlas.
...
+   apps in these environments. Alternatively, you 
+   can use the Web SDK for React Native or Node.js apps, but additional 
+   steps must be taken. For further instruction, visit the :ref:`Web SDK
```
Comment (mongodben) [inline source/web.txt:35]: > but additional steps must be taken.

avoid passive voice

File: source/web/install.txt [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+Overview
+--------
+
+This page shows you how to install the Realm Web SDK to use with your application. 
+
+Before You Begin
+----------------
+
+Before you begin installation, you'll need to :ref:`create an {+app+}<create-a-realm-app>` for
```
Comment (mongodben) [inline source/web/install.txt:21]: looks like we don't have the {+app+} source constant anymore. i believe nate removed last week. put in the relevant test.

this is causing the build to fail - https://github.com/mongodb/docs-realm/runs/7501851997?check_suite_focus=true

Comment (mongodben) [inline source/web/install.txt:21]:
```suggestion
Before you begin installation, you'll need to :ref:`create an App Services App<create-a-realm-app>` for 
```

File: source/web.txt [hunk 8]
```diff
@@ -27,11 +28,9 @@ Atlas.
...
-   <react-native-intro>`.
-
-.. button:: Create an Account
-   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_realm_web
+   There are SDKs built for :ref:`React Native <react-native-intro>`
+   and :ref:`Node.js <node-intro>` which you should use when creating
+   apps in these environments. Alternatively, you 
+   can use the Web SDK for React Native or Node.js apps, but you must take 
+   additional steps. For further instruction, visit the :ref:`Web SDK 
+   installation page <install>`.
```
Comment (mongodben) [inline source/web.txt:36]:
```suggestion
   can use the Web SDK for React Native or Node.js apps. For further instruction, refer to :ref:`Install for Node.js <install>`.
```

File: source/web/install.txt [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+      .. code-block:: html
+         
+         <script src="https://unpkg.com/realm-web/dist/bundle.iife.js"></script>
+      
+      Or import a specific version:
+
+      .. code-block:: html
+         
+         <script src="https://unpkg.com/realm-web@1.5.1/dist/bundle.iife.js"></script>
+
```
Comment (mongodben) [inline source/web/install.txt:61]:
```suggestion

.. _web-install-node

```

File: source/web/install.txt [hunk 10]
```diff
@@ -0,0 +1,70 @@
+.. _install:
```
Comment (mongodben) [inline source/web/install.txt:1]: rename this to be `web-install`. remember the refs are global for the repo

File: source/web/install.txt [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+      .. code-block:: html
+         
+         <script src="https://unpkg.com/realm-web@1.5.1/dist/bundle.iife.js"></script>
+
+Install for Node.js
+-------------------
+
+In order to use Realm Web for Node.js environments, you must add two 
+additional peer dependendencies during installation:
```
Comment (mongodben) [inline source/web/install.txt:66]:
```suggestion
In order to use Realm Web for Node.js environments, you must
install two additional peer dependencies:
```

File: source/web/install.txt [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+         
+         <script src="https://unpkg.com/realm-web/dist/bundle.iife.js"></script>
+      
+      Or import a specific version:
+
+      .. code-block:: html
+         
+         <script src="https://unpkg.com/realm-web@1.5.1/dist/bundle.iife.js"></script>
+
+.. _web-install-node
```
Comment (mongodben) [inline source/web/install.txt:62]: incorrect ref syntax
```suggestion
.. _web-install-node:
```

Comment (chloe-ratte) [reply source/web/install.txt:62]: OH that makes sense

**Review by mongodben — APPROVED** [review]
lgtm

**Pull request merged**